storm-client/src/jvm/org/apache/storm/utils/ConfigUtils.java

```diff
@@ -215,6 +215,20 @@ public static String absoluteStormLocalDir(Map<String, Object> conf) {
         }
     }
 
+    public static String absoluteStormBlobStoreDir(Map<String, Object> conf) {
+        String stormHome = System.getProperty("storm.home");
+        String blobStoreDir = (String) conf.get(Config.BLOBSTORE_DIR);
+        if (blobStoreDir == null) {
+            return null;
```

Comment:
I think we can make this a little nicer by having this return absoluteStormLocalDir. There's less risk of us accidentally forgetting to null check the result if this ever gets used elsewhere.